.github/actions/publish_to_pages_production/action.yml

```diff
@@ -0,0 +1,12 @@
+name: publish_to_pages_production
+description: Publish to cloudflare pages (production)
+runs:
+  using: composite
+  steps:
+  - name: Publish to cloudflare pages (production)
+    run: |-
+      npm i wrangler@3.1.0
+      cd packages/core
+      npx wrangler pages deploy dist/ --project-name=deriv-app-pages --branch=main
```

Comment:
https://stackoverflow.com/questions/70098241/using-secrets-in-composite-actions-github
Seems like we have pass tokens as inputs to a composite action

Reply:
thanks for the input @balakrishna-deriv this is applied on all the actions that require environment or secret.